README.md

```diff
@@ -1,13 +1,15 @@
 # tdr-terraform-github
 
-**Important Note**: tdr-terraform-github uses >= v1.9.8 of Terraform. Ensure that Terraform >= v1.9.8 is installed before proceeding.
+**Important Note**: tdr-terraform-github uses >= v1.12.2 of Terraform. Ensure that Terraform >= v1.12.2 is installed before proceeding.
 
 This repository contains the Terraform code to create the AWS resources needed to support the TDR Github Actions for the TDR Github repositories
 
 The Terraform is divided into two types:
 * *repository*: GitHub Actions repository level secrets and variables
 * *environment*: GitHub Actions environment level secrets and 
 
+There is no GitHub action to deploy this Terraform stack.  You need a laptop.
```

Comment:
Maybe re-word slightly: `... stack. Deployment should be done via local machine.`